source/StigData/Processed/Office-365ProPlus-2.3.xml

```diff
@@ -1315,7 +1317,7 @@ Use the Windows Registry Editor to navigate to the following key:
 
 HKCU\software\policies\microsoft\office\16.0\excel\options
 
-If value for disableautorepublishwarning is REG_DWORD = 1, this is not a finding.</RawString>
+If value for disableautorepublishwarning is REG_DWORD = 0, this is not a finding.</RawString>
```

Comment:
This is a hard coded rule that is no longer correct based on guidance.

Reply:
Hard coded rule:

V-223331::*::HardCodedRule(RegistryRule)@{DscResource = 'RegistryPolicyFile'; Key = 'HKEY_CURRENT_USER\Software\Policies\Microsoft\Office\16.0\Excel\Options'; ValueData = "1"; ValueName = 'disableautorepublishwarning'; ValueType = 'Dword'}